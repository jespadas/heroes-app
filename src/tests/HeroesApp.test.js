import React from 'react';
import { shallow } from 'enzyme';
import { HeroesApp } from '../HeroesApp';

describe('Test in <HeroesApp />', () => {

    let wrapper = shallow(<HeroesApp />);

    test('Should be displayed correctly', () => {

        expect(wrapper).toMatchSnapshot();

    })


})
