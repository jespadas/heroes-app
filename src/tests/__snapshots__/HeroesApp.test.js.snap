// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <HeroesApp /> Should be displayed correctly 1`] = `
<ContextProvider
  value={
    Object {
      "dispatch": [Function],
      "user": Object {
        "logged": false,
      },
    }
  }
>
  <AppRouter />
</ContextProvider>
`;
